shared/domains/chelonia/chelonia.js

```diff
@@ -641,6 +641,13 @@ export default (sbp('sbp/selectors/register', {
     }
     return this.pubsub
   },
+  'chelonia/handleEvent': async function (event: string) {
+    const { contractID } = GIMessage.deserializeHEAD(event)
+    if (self.registration) {
+      self.registration.showNotification('@@' + contractID, { body: sbp(this.config.stateSelector)[contractID] })
+    }
+    return await sbp('chelonia/private/in/enqueueHandleEvent', contractID, event)
+  },
```

Comment:
This asymmetry between incoming events via push notifications and incoming events over the websocket needs to either be addressed or at the very least commented.

Why is it that this selector was created? If it's just to show debugging like this, then IMO it would be better to delete it and call `'chelonia/private/in/enqueueHandleEvent'` directly in `self.addEventListener('push', function (event)` where this gets called.

That way there's no confusion about why these two different paths - and also why one selector is public while the other is private yet they both essentially do the exact same thing and only one of them is called on incoming event.

Reply:
> Why is it that this selector was created? 

We've discussed this, and I'm open to suggestions. Although debugging is easier this way, it's not the reason this was added. Rather, the reason is that the WS is owned and controlled by Chelonia and therefore it's possible to use a private selector.

On the other hand, the push channel can't be owned by Chelonia because it requires interacting with the browser in ways that I'd say are out of scope (see the 'push' event listener). Hence this selector, which can be thought of as a 'manual' way of handling events. This is similar to what we already do with `processMessage` and `ChatMain`.

Reply:
OK, then this is probably something that also needs to be explained and commented — that this selector primarily exists for ingesting events that come over through push notifications, which is something that the app would externally set up and manage and all.

(Also a reminder to not forget to remove this debugging info.)